app/controllers/api/services_controller.rb

```diff
@@ -18,6 +18,20 @@ def edit_resource(type, id, data)
       super(type, id, attributes)
     end
 
+    def add_resource_resource(type, id, data)
+      svc = resource_search(id, type, collection_class(type))
+      data['resources'].each do |resource_ref|
+        resource_type, resource_id = parse_href(resource_ref['href'])
+        resource = resource_search(resource_id, resource_type, collection_class(resource_type))
+        raise "Cannot assign #{resource_type} to #{service_ident(svc)}" unless resource.respond_to? :add_to_service
+        resource.add_to_service(svc)
```

Comment:
@mkanoor hm. I can see the case for that. What do you think about sending back an individual response to each resource instead? so the response will tell you which ones failed and which ones succeeded